src/main/java/com/arcbees/bourseje/server/services/VoteService.java

```diff
@@ -76,6 +82,32 @@ public void vote(VoteItem voteItem, String code) {
         voteItemDao.put(voteItem);
     }
 
+    public Collection<CandidateResult> getVotesPerCandidate() {
+        List<VoteItem> allVotes = voteItemDao.getAll();
```

Comment:
Let's take a look at that after everything is being done as an improvement. The business case is not critical enough to give real business value for that improvement at the moment.